data/hai missions.txt

```diff
@@ -1126,7 +1126,8 @@ mission "Nanachi 2"
 			choice
 				`	"Hey, what is going on?"`
 			`	You introduce yourself and the parents tell you that they've been on vacation here and need transport back to Hai-home. "Don't take this the wrong way, but captain Nanachi told us her ship is one of your human ships, a Heavy Shuttle was it? It has enough bunks for us but our sons already caused a ruckus on the way here in a Hai ship, we're worried that since they aren't used to the accomodations they might end up giving her too much trouble."`
-			`	You reassure them that the trip will be very quick, and that the ship's interior is somewhat similar to that of Hai ships, until they finally agree on going with her. They settle on the pay and you two help the Hai get their baggages to Nanachi's ship. While walking there you take a look at the prices for commodities in Hai-home, and tell Nanachi to stock up on heavy metals to sell there.`
+			`	You reassure them that the trip will be very quick, and that the ship's interior is somewhat similar to that of Hai ships, until they finally agree on going with her. They settle on the pay and you two help the Hai get their baggages to Nanachi's ship. When the family is focused on a conversation, you quietly ask Nanachi what was the issue she was having with the other people she encountered.`
+			`	"They all seemed willing to give me a job, but I ended up blowing it when I asked for more money, I just really need to get out of debt so I guess I went a bit overboard." she tries to play it cool, but it's clear that she is embarrased of losing so many potential jobs. When you reach her ship you take a look at the prices for commodities in Hai-home, and tell Nanachi to stock up on heavy metals to sell there, to help her make a bit more profit.`
```

Comment:
"baggage" not "baggages", we're not hobbitses.
Capitalize "she" in the last paragraph, after `overboard."`
The last sentence in the last paragraph is too long, try to break it up.